### PR TITLE
light and dark mode favicons

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #79** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/79?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/79?utm_source=stack-comment).

File: apps/next/public/favicon-dark/android-chrome-192x192.png
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply apps/next/public/favicon-dark/android-chrome-192x192.png:1]: I wasn't sure what we decided on here. I'll change it to white

File: apps/next/public/favicon-dark/safari-pinned-tab.svg
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply apps/next/public/favicon-dark/safari-pinned-tab.svg:1]: The only requirement for safari pinned tap is that it is monochrome

File: apps/next/public/favicon-dark/android-chrome-256x256.png
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply apps/next/public/favicon-dark/android-chrome-256x256.png:1]: Hm oddly enough the favicon generator didn't generate this one when I used it.

I read more into it and it seems that chrome gives priority to the 192x192 image. I'm removing the 256x256 entirely

**Review by 0xBigBoss — DISMISSED** [review]

File: apps/next/public/favicon-light/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-256x256.png",
```
Comment (0xBigBoss) [inline apps/next/public/favicon-light/site.webmanifest:11]: maybe delete this then?

Comment (youngkidwarrior) [reply apps/next/public/favicon-light/site.webmanifest]: Deleted and added a name and description to this

**Review by 0xBigBoss — APPROVED** [review]

**Pull request merged**